### PR TITLE
[PROGRAMMES-6746] added getType() and data content

### PR DESCRIPTION
**Pull request by tomwinnington**
Ticket:
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6746

Link:
http://sandbox.bbc.co.uk/programmes/profiles/z5gWDtzYM5nmgZD7zfgDFy/test-profile

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: config/services.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ parameters:
     app.idtquiz.service.class: App\ExternalApi\IdtQuiz\IdtQuizService
     app.favourites.baseurl: 'https://www.bbc.co.uk/modules/plugin/musicfavourite/init'
     app.favourites.service.class: App\ExternalApi\FavouritesButton\Service\FavouritesButtonService
-    app.isite.baseurl: 'https://api.live.bbc.co.uk/isite2-content-reader'
+    app.isite.baseurl: 'https://api.test.bbc.co.uk/isite2-content-reader'
```
Comment (Bobkidbob) [inline config/services.yaml]: Don't commit this.

File: src/ExternalApi/Isite/Domain/ContentBlock/AbstractContentBlock.php [hunk 2]
```diff
@@ -17,4 +17,9 @@ public function getTitle(): ?string
     {
         return $this->title;
     }
+
+    public function getType(): string
+    {
+
+    }
```
Comment (Bobkidbob) [inline src/ExternalApi/Isite/Domain/ContentBlock/AbstractContentBlock.php]: Does this empty method need to be here? If this is meant to be an abstract method make sure you implement it properly.

File: src/ExternalApi/Isite/Domain/ContentBlock/InteractiveActivity.php [hunk 3]
```diff
@@ -61,4 +61,10 @@ public function getHeight(): string
     {
         return $this->height;
     }
+
+    public function getType(): string
+    {
+        return 'interactiveActivity';
+
```
Comment (Bobkidbob) [inline src/ExternalApi/Isite/Domain/ContentBlock/InteractiveActivity.php]: Unnecessary whitespace.

**Comment by tomwinnington** [comment]
Test this please.
